packages/react/src/CTABanner/CTABanner.module.css

```diff
@@ -0,0 +1,82 @@
+.CTABanner {
+  position: relative;
+  max-width: 1280px;
+  margin: 0 auto;
+  display: flex;
+}
+
+.CTABanner--Shadow {
+  position: absolute;
+  transform: translate(-50%, 0%);
+  width: calc(60% + 60px);
+  height: calc(100% + 60px);
+  border-radius: 50% 22% 40% 80%;
+  filter: blur(50px);
+  top: 0%;
+  opacity: 0.7;
+}
+
+.CTABanner--StartShadow {
+  left: 30%;
+  background: radial-gradient(
```

Comment:
Shadow feels a bit faint compared to the Figma design. @danielguillan can you please help confirm we're using the right values here?

Reply:
That's correct. Building the shadow using two circular gradients is causing this problem. Additionally, the shadow doesn't extend properly around the corners, making these almost fade into the background.

![Comparison of the current shadow rendering vs. the Figma component](https://user-images.githubusercontent.com/175638/203407705-4cc25888-e6da-492d-b7ea-637303ff5124.png)

I think we can get pretty close to the rendering in Figma by using a single rectangular element and a linear gradient. I'm not sure if we've already tried something like this:

```css
.CTABanner::before {
  content: '';
  position: absolute;
  inset: var(--base-size-16);
  filter: blur(var(--base-size-96));
  transform: translateZ(0); /* Force GPU to fix blurring in Safari */
  background: linear-gradient(
    to right,
    hsla(var(--brand-CTABanner-shadow-color-start) / 1),
    hsla(var(--brand-CTABanner-shadow-color-end) / 1)
  ); /* We no longer need the alpha channel on the colors. */
}
```

_Note: I've used a pseudo-element here, but we can still use a dedicated element._


![Comparison of the current shadow rendering vs. the Figma component](https://user-images.githubusercontent.com/175638/203415450-61a3e606-c2e9-4533-af5f-29ee95b55b29.png)


It's not 1:1 with Figma (the colors are slightly duller and darker) but I think it's close enough.

Reply:
I will update it to this aapproach.